test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua

```diff
@@ -0,0 +1,198 @@
+    payload = '{ "audioStreamingIndicator" : "PAUSE" }'
+  }
+
+  self.mobileSession:Send(msg)
+
+  local startserviceEvent = Event()
+  startserviceEvent.matches =
+  function(_, data)
+    return ( (data.serviceType == 7) and (data.frameInfo == 2 or data.frameInfo == 3) )
+  end
+
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"}, {status = "UPDATING"}):Times(2)
+  EXPECT_HMICALL("BasicCommunication.PolicyUpdate")
+  :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator"):Times(0)
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: RPC SetAudioStreamingIndicator"):Times(0)
+
+  commonTestCases:DelayedExp(10000)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_PolicyTableUpdate_fails()
+  local startserviceEvent = Event()
+  startserviceEvent.matches =
+  function(_, data)
+    return ( (data.serviceType == 7) and (data.frameInfo == 2 or data.frameInfo == 3) )
+  end
+
+  local SystemFilesPath = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath")
+
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status="UPDATE_NEEDED"})
+
+  local RequestId_GetUrls = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+  EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS"} } )
+  :Do(function(_,_)
+      self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+        { requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"})
+      EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY"})
+      :Do(function()
+          local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", {requestType = "PROPRIETARY", fileName = "PolicyTableUpdate"},
+          "files/ptu_certificate_exist.json")
+          EXPECT_HMICALL("BasicCommunication.SystemRequest",{ requestType = "PROPRIETARY", fileName = SystemFilesPath.."/PolicyTableUpdate" })
+          :Do(function(_,_data1)
+              self.hmiConnection:SendResponse(_data1.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+              self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = SystemFilesPath.."/PolicyTableUpdate"})
+            end)
+          EXPECT_RESPONSE(CorIdSystemRequest, { success = true, resultCode = "SUCCESS"})
+        end)
+    end)
+
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: StartServiceNACK")
+  :ValidIf(function(_, data)
+      if data.frameInfo == 2 then
+        commonFunctions:printError("Service 7: StartServiceACK is received")
+        return false
+      elseif data.frameInfo == 3 then
+        print("Service 7: RPC NACK")
+        return true
+      else
+        commonFunctions:printError("Service 7: StartServiceACK/NACK is not received at all.")
+        return false
+      end
+    end)
+  :Timeout(20000)
+end
```

Comment:
@dboltovskyi, in this test using of this function can't be done because StartService is sent at previous test where PTU triggering is checked.